portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -74,11 +87,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String name = getParameter(request, "text-input-name", "");
     String content = getParameter(request, "text-input-content", "");
+    String imageUrl = getUploadedFileUrl(request, "imageUrl");
     long timestamp = System.currentTimeMillis();
 
     Entity commentEntity = new Entity("comment");
     commentEntity.setProperty("name", name);
     commentEntity.setProperty("content", content);
+    commentEntity.setProperty("imageUrl", imageUrl);
```

Comment:
Could you please verify that we don't get NullPointerException if imageUrl is null?

Reply:
Verified!